sympy/functions/combinatorial/numbers.py

```diff
@@ -1861,9 +1861,26 @@ def stirling(n, k, d=None, kind=2, signed=False):
 def _nT(n, k):
     """Return the partitions of ``n`` items into ``k`` parts. This
     is used by ``nT`` for the case when ``n`` is an integer."""
+    if k > n or k < 0 or n < 0:
```

Comment:
If `k > n` is false then `k <= n`. If also `k < 0` is false, then `k >= 0`. In that case, `n >= 0` by transitivity, so the third condition `n < 0` will always be false and could be omitted.

Reply:
Yeah, i will omit it.